### ISSUE TITLE
Scripts and Style links local rather than pulled from Internet

### ISSUE THREAD

**Issue by alligatorbait** [groveco/django-sql-explorer#411]
I have a great use case for an offline application to use django-sql-explorer and need the scripts and links all to be stored with the package for this purpose.

These were the references I found with a quick grep of the templates:
```
[explorer]# grep 'src="/' *
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/underscore.js/1.8.3/underscore-min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/list.js/1.5.0/list.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.36.0/codemirror.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.36.0/mode/sql/sql.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.36.0/addon/comment/comment.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/floatthead/2.1.1/jquery.floatThead.min.js"></script>
base.html:    <script src="//cdnjs.cloudflare.com/ajax/libs/pivottable/2.20.0/pivot.min.js"></script>

[explorer]# grep 'href="/' *
base.html:    <link href="//cdnjs.cloudflare.com/ajax/libs/codemirror/5.36.0/codemirror.min.css" rel="stylesheet" />
base.html:    <link href="//cdnjs.cloudflare.com/ajax/libs/pivottable/2.20.0/pivot.min.css" rel="stylesheet" />
fullscreen.html:    <link href="//cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
```

**Comment by marksweb** [comment]
I'm not keen on this. It'd be a big change to suit a very small number of users.

So if it's only a minority wanting locally served static files it be better done on an individual basis via overridden templates referencing the necessary files local to that project.

**Comment by alligatorbait** [comment]
Not an ideal scenario for a developer, but do-able. It would be a whole lot easier if those references we made more modular by having them in a separate {% include '....' %} file. How does that sound?

**Comment by spapas** [comment]
Hello a couple of things that may (or may not) help the discussion:

* There's a technique for getting the files from a CDN but falling back to local files if they are not found, it is rather simple and is decribed here https://stackoverflow.com/questions/1014203/best-way-to-use-googles-hosted-jquery-but-fall-back-to-my-hosted-library-on-go (so you can have best of both worlds)
* There was a post and a long HN discussion on why it's better to not use CDN: https://news.ycombinator.com/item?id=24745194

**Comment by marksweb** [comment]
@alligatorbait Bring all these static files local & make sure that they get pulled into the built package as part of #413

**Comment by alligatorbait** [comment]
@marksweb Are you sure? I just spotted that the one file already existed statically, so put in the change.

**Comment by marksweb** [comment]
@alligatorbait Yeah, most people won't actually notice the change really. If you put together the PR & make the changes to the manifest etc to make sure the files are pulled into the build then I don't mind so much because I don't have to do as much!! 👍

**Comment by alligatorbait** [comment]
Haha, alright. Good deal, and I'm happy to do the work, because it is less work to do it here than to eternally override/patch.

**Comment by alligatorbait** [comment]
I tested the changes with Django 3.1.2 and Python 3.6 with an egg file and ran the application with the chrome debugger panel to make sure no errors were introduced and to verify the new html was actually being used. Updated the documentation, and the manifest is an auto generated list of the explorer directory, so I didn't see any work to do there.

I'm thinking I got everything, but if not, please let me know and I'll make the necessary changes.

Thanks!

**Comment by marksweb** [comment]
Excellent thanks @alligatorbait 

I've also improved the way the app checks the blacklist so maybe we've got a 2.1 release already! 👍

**Comment by marksweb** [comment]
Closed by #413
